Flutter/example_superapp/ios/README.md

```diff
@@ -1,4 +1,4 @@
-# SuperApp Example 
+# Flutter iOS Super App Example 
 
 This document explains how to develop and integrate the functionality that provides the API for access to the Mini App Center, as well as the API for managing their cache, based on the `ExampleSuperApp.xcodeproj` example. 
```

Comment:
I would changed for: "functionality that provides the API for accessing the Mini App Center"